go/vt/vttablet/tabletserver/gc/tablegc.go

```diff
@@ -429,11 +430,33 @@ func (collector *TableGC) purge(ctx context.Context) (tableName string, err erro
 	// This saves a lot of load from the replication stream, avoiding excessive lags. It also
 	// avoids excessive IO on the replicas.
 	// (note that the user may skip the PURGE step if they want, but the step is on by default)
-	if _, err := conn.ExecuteFetch("SET sql_log_bin = OFF", 0, false); err != nil {
+
+	// However, disabling SQL_LOG_BIN requires SUPER privileges, and we don't know that we have that.
+	// e.g. on Amazon Aurora we do not have SUPER and cannot disable log bin.
```

Comment:
Nit: This is applicable to any database where we are running vttablet in externally managed mode - we might not have SUPER privileges even if it is MySQL. So I would re-word the comment to reflect that instead of naming Amazon Aurora.

Reply:
Reworded, but still kept `Amazon` as a known example